hr_timesheet_time_restriction/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 Dinar Gabbasov
+# Copyright 2022 Ooops404
+# Copyright 2022 Cetmix
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Time Restriction",
```

Comment:
HR Timesheet Sheet Restriction

Recomended, ambituous names are problematic, time restriction where? Here in the PR you have context but in OCA Apps, Odoo APPs o inside Odoo in APs the name is more relevant.